lib/galaxy/jobs/runners/kubernetes.py

```
@@ -487,6 +592,32 @@ def check_watched_item(self, job_state):
                 return None
             elif active > 0 and failed <= max_pod_retries:
                 if not job_state.running:
+                    job_state.running = True
+                    job_state.job_wrapper.change_state(model.Job.states.RUNNING)
+                    if job_state.job_wrapper.tool.tool_type == "interactive":
+                        k8s_job_prefix = self.__produce_k8s_job_prefix()
+                        k8s_job_name = "{}-{}".format(k8s_job_prefix, self.__force_label_conformity(job_state.job_wrapper.get_id_tag()))
+                        log.debug(f'Configuring entry points and deploying service/ingress for job with ID {job_state.job_id}')
+                        k8s_service_obj = service_object_dict(
+                            self.runner_params,
+                            k8s_job_name,
+                            self.__get_k8s_service_spec(job_state)
+                        )
+                        k8s_ingress_obj = ingress_object_dict(
+                            self.runner_params,
+                            k8s_job_name,
+                            self.__get_k8s_ingress_spec(job_state)
+                        )
+                        service = Service(self._pykube_api, k8s_service_obj)
+                        try:
+                            service.create()
+                        except Exception as e:
+                            log.debug(f'Failed to create service k8s resource for IT due to: {str(e)}')
```

Are these exception fatal for running an interactive tool ? In that case I'd re-raise the exception (or maybe not even catch them?)